numpy/testing/_private/utils.py

```diff
@@ -813,7 +814,7 @@ def func_assert_same_pos(x, y, func=isnan, hasval='nan'):
                     if all(~nonzero):
                         max_rel_error = array(inf)
                     else:
-                        max_rel_error = max(error[nonzero] / abs(y[nonzero]))
+                        max_rel_error = np.max(error[nonzero] / np.abs(y[nonzero]))
```

Comment:
I don't understand why this changes fixes anything - do the tests below fail without it?

Reply:
I think this whole patch might have become defunct via a rebase

Reply:
seems so